app/src/main/java/com/locked/shingranicommunity/event/EventListViewModel.kt

```diff
@@ -131,15 +134,16 @@ class EventListViewModel @Inject constructor(
             eventItem.address?.let {data.showMap.value = true}
         }
 
+        @RequiresApi(Build.VERSION_CODES.O)
         private fun initDateTime() {
             val timeStr = eventItem.time ?: ""
-            val fromFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.getDefault())
+            val fromFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssz", Locale.getDefault())
             val toDateFormat = SimpleDateFormat("E, MMM dd", Locale.getDefault())
             val toTimeFormat = SimpleDateFormat("hh:mm", Locale.getDefault())
             try {
-                val dt: Date = fromFormat.parse(timeStr)
-                data.date.value = toDateFormat.format(dt)
-                data.time.value = toTimeFormat.format(dt)
+//                val dt: Date = fromFormat.parse(instant.toString())
+                data.date.value = timeStr.split("T")[0]
+                data.time.value = timeStr.split("T")[1]
```

Comment:
revert this code

Reply:
but that is not working because we have send to backend formated date and time and its a string and the format is not parsing it give exception unable to parsing 
so since we know we have always spacific string date time just a T is dividing it we can use split string